frontend/src/plugin/pluginsSlice.ts

```diff
@@ -90,14 +90,17 @@ export type PluginInfo = {
 export interface PluginsState {
   /** Have plugins finished executing? */
   loaded: boolean;
+  /** Information for the plugin's enable */
```

Comment:
a better comment would be something like `Map where key is plugin's name and value is whethere it is enabled`

Reply:
fixed!